teamA/leeHyeonTae/.env.sample

```
@@ -0,0 +1,11 @@
+DATABASE_URL="rdbms://username:password@host:db_port/db_name" 
+
+PORT= port
+
+TYPEORM_CONNECTION=rdbms 
+TYPEORM_HOST=host 
+TYPEORM_USERNAME=username 
+TYPEORM_PASSWORD='password'
+TYPEORM_DATABASE=db_name 
+TYPEORM_PORT=db_port 
+TYPEORM_LOGGING= logging 여부
```

logging 여부는 true 또는 false 값을 갖습니다.

명확한 값을 넣을 수 있도록 해당 부분의 예시를 다음과 같이 바꿔주세요!
```suggestion
TYPEORM_LOGGING= boolean (혹은BOOLEAN)
```